README.md

```diff
@@ -252,12 +264,33 @@ Example:
 <script src='https://cdn.jsdelivr.net/npm/@clappr/stats-plugin@latest/dist/clappr-stats.min.js'></script>
 ```
 
+```javascript
+{
+  plugins: {
+    container: [ClapprStats],
+    playback: [HlsjsPlayback],
+  }
+}
+```
+
+You can pass plugins of any category in on flat array too. Example:
+
 ```javascript
 {
   plugins: [ClapprStats, HlsjsPlayback]
 }
 ```
 
+#### plugins.externalPluginsLoadPrecedence
+> Default Value: `true`
+
+Force external plugins to be loaded after default Clappr plugins.
+
+#### plugins.externalPlaybacksLoadPrecedence
+> Default Value: `true`
+
+Force external playbacks to be loaded after default Clappr playbacks.
```

Comment:
I guess we need to change that `after` behavior description, right?

Reply:
Sure! Done in this commit d553cfa75ad84ef97abd7c4e93205c3509413d27